extension/chrome/elements/compose-modules/compose-attachments-module.ts

```diff
@@ -32,13 +32,16 @@ export class ComposeAttachmentsModule extends ViewModule<ComposeView> {
   };
 
   private getMaxAttachmentSizeAndOversizeNotice = async (): Promise<AttachmentLimits> => {
-    const sizeMb = 25;
+    // To prevent size bloating, it is recommended to set the attachment size limit to 19MB for Rich-text editor mode - https://github.com/FlowCrypt/flowcrypt-browser/issues/2538#issuecomment-1639926581
+    const sizeMb = this.view.sendBtnModule.popover.choices.richtext ? 19 : 25;
```

Comment:
let's use `this.view.inputModule.isRichText()` instead `this.view.sendBtnModule.popover.choices.richtext`, so it'll be consistent with other code

Reply:
Thank you - I haven't spotted the predefined function too early 😅, I will use it now.